packages/adapter-nuxt2/test/plugin-project-init.test.ts

```diff
@@ -180,3 +180,39 @@ test("Slice Simulator page file is not formatted if formatting is disabled", asy
 		}),
 	);
 });
+
+test("migrates and configures nuxt exporting a function", async (ctx) => {
```

Comment:
I think a better name for the test would be "warns user that Nuxt config could not be updated", not sure this either migrate or configure anything here 🤔

Reply:
oh, I forgot to update that one :)